src/presentation/http/router/join.test.ts

```diff
@@ -3,16 +3,62 @@ import { describe, test, expect } from 'vitest';
 describe('Join API', () => {
   describe('POST /join/:hash', () => {
     test('Returns 406 when user is already in the team', async () => {
-      const hash = 'Hzh2hy4igf';
-      const userId = 2;
+      const invitationHash = 'Hzh2hy4igf';
+      const userId = 1;
       const accessToken = global.auth(userId);
 
-      const response = await global.api?.fakeRequest({
+      test.todo('get rid of restarting database data in tests (use restart-database-data script in beforeEach)');
+      // truncate all tables, which are needed
+      await global.db.query(`TRUNCATE public.users, public.notes, public.note_settings CASCADE`);
+
+      // restart autoincrement sequences for data to start with id 1
+      await global.db.query(`ALTER sequence users_id_seq RESTART WITH 1`);
+      await global.db.query(`ALTER sequence notes_id_seq RESTART WITH 1`);
+      await global.db.query(`ALTER sequence note_settings_id_seq RESTART WITH 1`);
```

Comment:
I'd suggest moving it to the util as well.

Reply:
➕

Reply:
done